mm/kfence/report.c

```diff
@@ -131,7 +131,8 @@ void kfence_report_error(unsigned long address, const struct kfence_metadata *me
 	int num_stack_entries = stack_trace_save(stack_entries, KFENCE_STACK_DEPTH, 1);
 	int skipnr = get_stack_skipnr(stack_entries, num_stack_entries, type);
 
-	lockdep_assert_held(&meta->lock);
+	if (meta)
```

Comment:
This isn't the only place dereferencing `meta` in this function, shall we do anything about the rest?

Reply:
Only for OOB it's non-optional. I added a precondition check.